apis/onlyfans/onlyfans.py

```diff
@@ -11,25 +12,37 @@
 from mergedeep import merge, Strategy
 import jsonpickle
 import copy
+from random import random
 
 
 # Zero reason to do this lmao
 
 
-def create_sign(session, link, sess, user_agent, text="onlyfans"):
+def create_sign(session, link, sess, x_bc, text="onlyfans"):
     # Users: 300000 | Creators: 301000
-    time2 = str(int(round(time.time() * 1000-301000)))
+    time2 = str(int(round(time.time())))
     path = urlparse(link).path
     query = urlparse(link).query
-    path = path+"?"+query
-    a = [sess, time2, path, user_agent, text]
+    path = path if not query else f"{path}?{query}"
+    static_param = "BcsLYSyemJCNrob8u6QWziudT5Xx4LlO"
+    a = [static_param, time2, path, "0"]
```

Comment:
[@hippothon mentioned that the zero should be the user's ID](https://github.com/DIGITALCRIMINAL/OnlyFans/issues/1078#issuecomment-830679779), unless not logged in. Is it confirmed working with zero?

Reply:
Actually I can proceed to video download stage so it should be working